.github/workflows/analyze.yml

```diff
@@ -2,7 +2,7 @@ name: Analyze
 on: [push]
 jobs:
     analyze:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-22.04
```

Comment:
If 24.04 is available, I think we should use that, as it's the latest LTS:
https://ubuntu.com/about/release-cycle